pages/resources/blog/[slug].jsx

```diff
@@ -57,7 +65,9 @@ export const getStaticProps = async ({ params }) => {
 export const getStaticPaths = async () => {
   let postPaths;
   try {
-    postPaths = await prisma.post.findMany();
+    // @P_ERR
+    // postPaths = await prisma.post.findMany();
```

Comment:
what is the specific error here? this is not using our database but relying on our cache so just want to be sure i forgot to add this in earlier

Reply:
im sure its because i keep updating things like vulnerabilities but just curious if you could screenshot or smth

Reply:
Yep, here's my browser's screen,
https://gyazo.com/dd31a8773c9c771cf60c427477847645

This is the terminal's,
https://gyazo.com/b9591bcecad01bf3239332ca6bce2fb3
https://gyazo.com/668c1b53fadd231d0583e32020b9c1f1

Reply:
oh this is because you don't have the `.env` file - I have some time to dig it up tmrw